{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "3460f797",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting pymysql\n",
      "  Downloading PyMySQL-1.0.2-py3-none-any.whl (43 kB)\n",
      "\u001b[K     |████████████████████████████████| 43 kB 1.1 MB/s eta 0:00:01\n",
      "\u001b[?25hInstalling collected packages: pymysql\n",
      "Successfully installed pymysql-1.0.2\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install pymysql"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "9c3f78cf",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting mysqlclient\n",
      "  Downloading mysqlclient-2.1.0.tar.gz (87 kB)\n",
      "\u001b[K     |████████████████████████████████| 87 kB 1.5 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_b5014caba480499d8f6a050b6a7e8138/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_b5014caba480499d8f6a050b6a7e8138/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-6blq6m38\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_b5014caba480499d8f6a050b6a7e8138/\n",
      "    Complete output (15 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_b5014caba480499d8f6a050b6a7e8138/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_b5014caba480499d8f6a050b6a7e8138/setup_posix.py\", line 70, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_b5014caba480499d8f6a050b6a7e8138/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    mysql_config --version\n",
      "    mariadb_config --version\n",
      "    mysql_config --libs\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/de/79/d02be3cb942afda6c99ca207858847572e38146eb73a7c4bfe3bdf154626/mysqlclient-2.1.0.tar.gz#sha256=973235686f1b720536d417bf0a0d39b4ab3d5086b2b6ad5e6752393428c02b12 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.3.tar.gz (88 kB)\n",
      "\u001b[K     |████████████████████████████████| 88 kB 5.0 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_12cd79cfe2834565bb403ecb20dd825d/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_12cd79cfe2834565bb403ecb20dd825d/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-aja18yw5\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_12cd79cfe2834565bb403ecb20dd825d/\n",
      "    Complete output (15 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_12cd79cfe2834565bb403ecb20dd825d/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_12cd79cfe2834565bb403ecb20dd825d/setup_posix.py\", line 70, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_12cd79cfe2834565bb403ecb20dd825d/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    mysql_config --version\n",
      "    mariadb_config --version\n",
      "    mysql_config --libs\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/3c/df/59cd2fa5e48d0804d213bdcb1acb4d08c403b61c7ff7ed4dd4a6a2deb3f7/mysqlclient-2.0.3.tar.gz#sha256=f6ebea7c008f155baeefe16c56cd3ee6239f7a5a9ae42396c2f1860f08a7c432 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.2.tar.gz (88 kB)\n",
      "\u001b[K     |████████████████████████████████| 88 kB 4.1 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1bd97d0d0a6e4f6db3a8eab1e9ac411b/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1bd97d0d0a6e4f6db3a8eab1e9ac411b/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-pgnd2ucl\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1bd97d0d0a6e4f6db3a8eab1e9ac411b/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1bd97d0d0a6e4f6db3a8eab1e9ac411b/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1bd97d0d0a6e4f6db3a8eab1e9ac411b/setup_posix.py\", line 65, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1bd97d0d0a6e4f6db3a8eab1e9ac411b/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/0e/68/f79de0e0fcdb041783124b825771e0ba6a3d0893b7b4c4735f130e42af47/mysqlclient-2.0.2.tar.gz#sha256=8df057b08fc27d8f7106bfa997d0a21e2acef017f905f06d6fb0aa6a20d4d2b2 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.1.tar.gz (87 kB)\n",
      "\u001b[K     |████████████████████████████████| 87 kB 4.9 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7e070bc2171e4b138e86a9f33aed4800/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7e070bc2171e4b138e86a9f33aed4800/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-h2_c6ik6\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7e070bc2171e4b138e86a9f33aed4800/\n",
      "    Complete output (12 lines):\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7e070bc2171e4b138e86a9f33aed4800/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7e070bc2171e4b138e86a9f33aed4800/setup_posix.py\", line 65, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7e070bc2171e4b138e86a9f33aed4800/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/a5/e1/e5f2b231c05dc51d9d87fa5066f90d1405345c54b14b0b11a1c859020f21/mysqlclient-2.0.1.tar.gz#sha256=fb2f75aea14722390d2d8ddf384ad99da708c707a96656210a7be8af20a2c5e5 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.0.tar.gz (87 kB)\n",
      "\u001b[K     |████████████████████████████████| 87 kB 3.6 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6ff45e1acbc34930b342cefa25e1060a/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6ff45e1acbc34930b342cefa25e1060a/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-jwqoel4a\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6ff45e1acbc34930b342cefa25e1060a/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6ff45e1acbc34930b342cefa25e1060a/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6ff45e1acbc34930b342cefa25e1060a/setup_posix.py\", line 65, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6ff45e1acbc34930b342cefa25e1060a/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/ca/cf/dd8124bf8b3d2ae465db9c8ce35c51650c31784b095f41faf6f8cad987a0/mysqlclient-2.0.0.tar.gz#sha256=201dfcc2a3c8a0859ea399947f1af8abe74b706beba0ebac149180d9d7b0f6e8 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.6.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 3.1 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7c2faaa084904e6681be00cbb85ab9c0/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7c2faaa084904e6681be00cbb85ab9c0/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-4hvctjr6\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7c2faaa084904e6681be00cbb85ab9c0/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7c2faaa084904e6681be00cbb85ab9c0/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7c2faaa084904e6681be00cbb85ab9c0/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_7c2faaa084904e6681be00cbb85ab9c0/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/d0/97/7326248ac8d5049968bf4ec708a5d3d4806e412a42e74160d7f266a3e03a/mysqlclient-1.4.6.tar.gz#sha256=f3fdaa9a38752a3b214a6fe79d7cae3653731a53e577821f9187e67cbecb2e16 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.5.tar.gz (86 kB)\n",
      "\u001b[K     |████████████████████████████████| 86 kB 4.3 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_5c185d9a10c54a1f94f25a50705f2f7a/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_5c185d9a10c54a1f94f25a50705f2f7a/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-bccko5vk\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_5c185d9a10c54a1f94f25a50705f2f7a/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_5c185d9a10c54a1f94f25a50705f2f7a/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_5c185d9a10c54a1f94f25a50705f2f7a/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_5c185d9a10c54a1f94f25a50705f2f7a/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f8/9b/5db9a03e2088a87c26e3e4d4c7f7e8f4c2dbae610f9521cdfac15755a795/mysqlclient-1.4.5.tar.gz#sha256=e80109b0ae8d952b900b31b623181532e5e89376d707dcbeb63f99e69cefe559 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.4.4.tar.gz (86 kB)\n",
      "\u001b[K     |████████████████████████████████| 86 kB 4.7 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_07a0318f0d4941a3bb5db855e0afa51e/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_07a0318f0d4941a3bb5db855e0afa51e/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-h_0cxq8i\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_07a0318f0d4941a3bb5db855e0afa51e/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_07a0318f0d4941a3bb5db855e0afa51e/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_07a0318f0d4941a3bb5db855e0afa51e/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_07a0318f0d4941a3bb5db855e0afa51e/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/4d/38/c5f8bac9c50f3042c8f05615f84206f77f03db79781db841898fde1bb284/mysqlclient-1.4.4.tar.gz#sha256=9c737cc55a5dc8dd3583a942d5a9b21be58d16f00f5fefca4e575e7d9682e98c (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.3.tar.gz (86 kB)\n",
      "\u001b[K     |████████████████████████████████| 86 kB 3.3 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_90fbb3d33cb845cc854ba7bdefac9172/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_90fbb3d33cb845cc854ba7bdefac9172/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-lpqy7blc\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_90fbb3d33cb845cc854ba7bdefac9172/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    /bin/sh: mariadb_config: command not found\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_90fbb3d33cb845cc854ba7bdefac9172/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_90fbb3d33cb845cc854ba7bdefac9172/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_90fbb3d33cb845cc854ba7bdefac9172/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/13/37/6c941aabeb78464a538b7d43266bfae645993779e3ca3e12fe82b4590f52/mysqlclient-1.4.3.tar.gz#sha256=0d3eeb97534c9f42e9b65bd35c1a35fadc2f6d7ea5d9fdbda5339a569156344b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.2.post1.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 4.2 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_4da323d751a34f0ebebd7968fd9c19de/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_4da323d751a34f0ebebd7968fd9c19de/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-yax6j08a\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_4da323d751a34f0ebebd7968fd9c19de/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_4da323d751a34f0ebebd7968fd9c19de/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_4da323d751a34f0ebebd7968fd9c19de/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_4da323d751a34f0ebebd7968fd9c19de/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f4/f1/3bb6f64ca7a429729413e6556b7ba5976df06019a5245a43d36032f1061e/mysqlclient-1.4.2.post1.tar.gz#sha256=f257d250f2675d0ef99bd318906f3cfc05cef4a2f385ea695ff32a3f04b9f9a7 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.2.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 2.2 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_dd98dea812994320bd0a1a115f312640/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_dd98dea812994320bd0a1a115f312640/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-44eba4ve\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_dd98dea812994320bd0a1a115f312640/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_dd98dea812994320bd0a1a115f312640/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_dd98dea812994320bd0a1a115f312640/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_dd98dea812994320bd0a1a115f312640/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/73/79/67ddf24ac31b05d741f0ac87fa612c7a11bab1b39b8270ed7344d149f8af/mysqlclient-1.4.2.tar.gz#sha256=b95edaa41d6cc47deecabcdcbb5ab437ad9ae6d8955f5cf10d1847b37e66ef5e (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.4.1.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 3.5 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_467ad8638c7a4db0943340b6657433d1/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_467ad8638c7a4db0943340b6657433d1/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-5zlr5za5\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_467ad8638c7a4db0943340b6657433d1/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_467ad8638c7a4db0943340b6657433d1/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_467ad8638c7a4db0943340b6657433d1/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_467ad8638c7a4db0943340b6657433d1/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/de/d7/919042fb7ed78a89e21c4e43d5daed90d5df18ec183bae3de493a7d9e3ed/mysqlclient-1.4.1.tar.gz#sha256=a62220410e26ce2d2ff94dd0138c3ecfb91db634464a9afb4c8e6b50f0a67e00 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.0.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 3.7 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8db6fd32a8c54e20bdb3e0bd71c354ad/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8db6fd32a8c54e20bdb3e0bd71c354ad/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-njc9dbsa\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8db6fd32a8c54e20bdb3e0bd71c354ad/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8db6fd32a8c54e20bdb3e0bd71c354ad/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8db6fd32a8c54e20bdb3e0bd71c354ad/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8db6fd32a8c54e20bdb3e0bd71c354ad/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/05/73/fac741eef89b0257fb1807de80e96474f40d52739cdbc32c90c4a793e79d/mysqlclient-1.4.0.tar.gz#sha256=efeaacd04feea7d44fef2a5adf4df335b438458f8b89144307dc5a90025761a5 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.14.tar.gz (91 kB)\n",
      "\u001b[K     |████████████████████████████████| 91 kB 4.7 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_495882b7050848489d7cfe0148ad4e0e/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_495882b7050848489d7cfe0148ad4e0e/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-rwwf4nrq\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_495882b7050848489d7cfe0148ad4e0e/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_495882b7050848489d7cfe0148ad4e0e/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_495882b7050848489d7cfe0148ad4e0e/setup_posix.py\", line 53, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_495882b7050848489d7cfe0148ad4e0e/setup_posix.py\", line 28, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f7/a2/1230ebbb4b91f42ad6b646e59eb8855559817ad5505d81c1ca2b5a216040/mysqlclient-1.3.14.tar.gz#sha256=3981ae9ce545901a36a8b7aed76ed02960a429f75dc53b7ad77fb2f9ab7cd56b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.13.tar.gz (90 kB)\n",
      "\u001b[K     |████████████████████████████████| 90 kB 4.3 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6fe04b0cedc94e3891ff02d34c412e4d/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6fe04b0cedc94e3891ff02d34c412e4d/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-u7wh0_z4\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6fe04b0cedc94e3891ff02d34c412e4d/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6fe04b0cedc94e3891ff02d34c412e4d/setup.py\", line 18, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6fe04b0cedc94e3891ff02d34c412e4d/setup_posix.py\", line 53, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_6fe04b0cedc94e3891ff02d34c412e4d/setup_posix.py\", line 28, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/ec/fd/83329b9d3e14f7344d1cb31f128e6dbba70c5975c9e57896815dbb1988ad/mysqlclient-1.3.13.tar.gz#sha256=ff8ee1be84215e6c30a746b728c41eb0701a46ca76e343af445b35ce6250644f (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.12.tar.gz (89 kB)\n",
      "\u001b[K     |████████████████████████████████| 89 kB 2.9 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_eee9e10e0f054a1881bd44eda3700287/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_eee9e10e0f054a1881bd44eda3700287/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-x6rximqc\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_eee9e10e0f054a1881bd44eda3700287/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_eee9e10e0f054a1881bd44eda3700287/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_eee9e10e0f054a1881bd44eda3700287/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_eee9e10e0f054a1881bd44eda3700287/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6f/86/bad31f1c1bb0cc99e88ca2adb7cb5c71f7a6540c1bb001480513de76a931/mysqlclient-1.3.12.tar.gz#sha256=2d9ec33de39f4d9c64ad7322ede0521d85829ce36a76f9dd3d6ab76a9c8648e5 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.11.tar.gz (89 kB)\n",
      "\u001b[K     |████████████████████████████████| 89 kB 3.6 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8d2bb822a3a44abc9ce5c49013694526/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8d2bb822a3a44abc9ce5c49013694526/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-yaodfeoo\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8d2bb822a3a44abc9ce5c49013694526/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8d2bb822a3a44abc9ce5c49013694526/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8d2bb822a3a44abc9ce5c49013694526/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_8d2bb822a3a44abc9ce5c49013694526/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6a/4d/856d1848e9b7fa8577a31c0188592f472e5cbc94f588c2a40ff930533d73/mysqlclient-1.3.11.tar.gz#sha256=9a79b38843d61f651266ff16b9fdcc8830d21f0fb852e4406a19e9a715bdf3e2 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.10.tar.gz (82 kB)\n",
      "\u001b[K     |████████████████████████████████| 82 kB 981 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_bfdc98f848644c7ea4393968c46d42ba/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_bfdc98f848644c7ea4393968c46d42ba/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-zhde92jf\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_bfdc98f848644c7ea4393968c46d42ba/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_bfdc98f848644c7ea4393968c46d42ba/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_bfdc98f848644c7ea4393968c46d42ba/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_bfdc98f848644c7ea4393968c46d42ba/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/40/9b/0bc869f290b8f49a99b8d97927f57126a5d1befcf8bac92c60dc855f2523/mysqlclient-1.3.10.tar.gz#sha256=452aaf9d2ba1a4fb85c134c88f9b1857852deed63b3b13255d7a65d2c1297262 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.9.tar.gz (81 kB)\n",
      "\u001b[K     |████████████████████████████████| 81 kB 4.6 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_f2f9f2aa48bd40358c32454b698882fb/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_f2f9f2aa48bd40358c32454b698882fb/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-j0ukqs49\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_f2f9f2aa48bd40358c32454b698882fb/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_f2f9f2aa48bd40358c32454b698882fb/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_f2f9f2aa48bd40358c32454b698882fb/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_f2f9f2aa48bd40358c32454b698882fb/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/db/f5/c8e1657985c31dda16e434edf5257c31572fa5faacd7e48b1618390e4b18/mysqlclient-1.3.9.tar.gz#sha256=990ccf1e1f15b9a291b811c993dc1c216566714bc14e6581b38665bd61c28c99 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.8.tar.gz (81 kB)\n",
      "\u001b[K     |████████████████████████████████| 81 kB 4.3 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1f3d6465e4d649b79098119a65320da9/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1f3d6465e4d649b79098119a65320da9/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-0b8_opno\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1f3d6465e4d649b79098119a65320da9/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1f3d6465e4d649b79098119a65320da9/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1f3d6465e4d649b79098119a65320da9/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_1f3d6465e4d649b79098119a65320da9/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/31/ce/7ea049b3d5929b3cce3104967ccca218a9f054517f0d15dfdaf4e76da2a0/mysqlclient-1.3.8.tar.gz#sha256=c64bc386c0b771ed6ff76de0f52e0f957f98d9e13c83b0ddb821afdbe3f5d970 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.7.tar.gz (79 kB)\n",
      "\u001b[K     |████████████████████████████████| 79 kB 4.9 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_08b79146b9e849fe8e55b414a1f46143/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_08b79146b9e849fe8e55b414a1f46143/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-r9z2yobh\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_08b79146b9e849fe8e55b414a1f46143/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_08b79146b9e849fe8e55b414a1f46143/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_08b79146b9e849fe8e55b414a1f46143/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_08b79146b9e849fe8e55b414a1f46143/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/74/ff/4e964e20b559e55d7afa60fbccc6a560f2adf289813bd3d7eb4eb8a87093/mysqlclient-1.3.7.tar.gz#sha256=c74a83b4cb2933d0e43370117eeebdfa03077ae72686d2df43d31879267f1f1b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.6.tar.gz (78 kB)\n",
      "\u001b[K     |████████████████████████████████| 78 kB 4.2 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_288e7c60f5d2449a88c1323ba6e1fa1e/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_288e7c60f5d2449a88c1323ba6e1fa1e/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-uy4f012x\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_288e7c60f5d2449a88c1323ba6e1fa1e/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_288e7c60f5d2449a88c1323ba6e1fa1e/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_288e7c60f5d2449a88c1323ba6e1fa1e/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_288e7c60f5d2449a88c1323ba6e1fa1e/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/09/29/6648563af4b45798ba667f17ea55166d7b0ed8717937c06977a1fdbe7df6/mysqlclient-1.3.6.tar.gz#sha256=1f63a8ab111530a0ff9acaeb118074f11e7564b7c124f2e7fc56fcac641aecf2 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.5.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 4.1 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_9e96a2a3041241b78dc648a9a1bea533/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_9e96a2a3041241b78dc648a9a1bea533/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-g3e40ec2\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_9e96a2a3041241b78dc648a9a1bea533/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_9e96a2a3041241b78dc648a9a1bea533/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_9e96a2a3041241b78dc648a9a1bea533/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_9e96a2a3041241b78dc648a9a1bea533/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/82/ea/336daf6c7baa564d0c434ab748218d1a7dc61dfc7066e98b72afd24fd2b9/mysqlclient-1.3.5.tar.gz#sha256=80ea68f6f3c2cfbf35c91adde2b40de1f2c31132118d2dcee048c72423f678bf (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.4.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 4.2 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_de3e34ce55f64d70b5ea2d04cc253d0f/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_de3e34ce55f64d70b5ea2d04cc253d0f/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-egp04l1m\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_de3e34ce55f64d70b5ea2d04cc253d0f/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_de3e34ce55f64d70b5ea2d04cc253d0f/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_de3e34ce55f64d70b5ea2d04cc253d0f/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_de3e34ce55f64d70b5ea2d04cc253d0f/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/17/d9/03b08e6a033401335b720806b87596aee8d7f7aa87539713238905c4a8d1/mysqlclient-1.3.4.tar.gz#sha256=a0263383f8ac33ad12d68dc61fa8ccbee41244ce916287c6cee71bf8aea82c5c (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.3.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 4.5 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ae4867c7fd1d4f64ae3ef75822c90ef5/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ae4867c7fd1d4f64ae3ef75822c90ef5/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-rdwscuvf\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ae4867c7fd1d4f64ae3ef75822c90ef5/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ae4867c7fd1d4f64ae3ef75822c90ef5/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ae4867c7fd1d4f64ae3ef75822c90ef5/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ae4867c7fd1d4f64ae3ef75822c90ef5/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/26/53/c575db342bfca0213c9243ed2642bfadcd420b18d2f477dc812543e3d83a/mysqlclient-1.3.3.tar.gz#sha256=0f0887882d71bf2a079d9724d9323053cfbee8ab2dcfc989daf97d7ce5be9af9 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.2.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 4.1 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_33138fe0e14d46df9bc4a7ea867d1aa8/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_33138fe0e14d46df9bc4a7ea867d1aa8/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-vs_mfm0s\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_33138fe0e14d46df9bc4a7ea867d1aa8/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_33138fe0e14d46df9bc4a7ea867d1aa8/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_33138fe0e14d46df9bc4a7ea867d1aa8/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_33138fe0e14d46df9bc4a7ea867d1aa8/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/2d/df/5440ee86bbb248325cdd6e1fc9cbbba365018a0d2d57f673610e020e6b1d/mysqlclient-1.3.2.tar.gz#sha256=e44025830501b9f70f8c2fe8eeff66f0df2df198802f7295801dac199b8236ef (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.1.tar.gz (76 kB)\n",
      "\u001b[K     |████████████████████████████████| 76 kB 4.1 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ea59c896982f487f879e17be38e779dd/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ea59c896982f487f879e17be38e779dd/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-tbz80aix\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ea59c896982f487f879e17be38e779dd/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ea59c896982f487f879e17be38e779dd/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ea59c896982f487f879e17be38e779dd/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_ea59c896982f487f879e17be38e779dd/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6b/ba/4729d99e85a0a35bb46d55500570de05b4af10431cef174b6da9f58a0e50/mysqlclient-1.3.1.tar.gz#sha256=3549e8a61f10c8cd8eac6581d3f44d0594f535fb7b29e6090db3a0bc547b25ad (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.0.tar.gz (76 kB)\n",
      "\u001b[K     |████████████████████████████████| 76 kB 10.5 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_0d2d2911ba964d8487b0b007168863e3/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_0d2d2911ba964d8487b0b007168863e3/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-tz4rr_c1\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_0d2d2911ba964d8487b0b007168863e3/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: mysql_config: command not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_0d2d2911ba964d8487b0b007168863e3/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_0d2d2911ba964d8487b0b007168863e3/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-b6c77o9g/mysqlclient_0d2d2911ba964d8487b0b007168863e3/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6a/91/bdfe808fb5dc99a5f65833b370818161b77ef6d1e19b488e4c146ab615aa/mysqlclient-1.3.0.tar.gz#sha256=06eb5664e3738b283ea2262ee60ed83192e898f019cc7ff251f4d05a564ab3b7 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[31mERROR: Could not find a version that satisfies the requirement mysqlclient (from versions: 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5, 1.3.6, 1.3.7, 1.3.8, 1.3.9, 1.3.10, 1.3.11rc1, 1.3.11, 1.3.12, 1.3.13, 1.3.14, 1.4.0rc1, 1.4.0rc2, 1.4.0rc3, 1.4.0, 1.4.1, 1.4.2, 1.4.2.post1, 1.4.3, 1.4.4, 1.4.5, 1.4.6, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0rc1, 2.1.0)\u001b[0m\n",
      "\u001b[31mERROR: No matching distribution found for mysqlclient\u001b[0m\n",
      "\u001b[?25hNote: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install mysqlclient"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "ff3663d0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting MySQL-python\n",
      "  Downloading MySQL-python-1.2.5.zip (108 kB)\n",
      "\u001b[K     |████████████████████████████████| 108 kB 1.6 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_66562a10f1fb4d89acbe3d321844b8cd/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_66562a10f1fb4d89acbe3d321844b8cd/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-284v29o9\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_66562a10f1fb4d89acbe3d321844b8cd/\n",
      "    Complete output (7 lines):\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_66562a10f1fb4d89acbe3d321844b8cd/setup.py\", line 13, in <module>\n",
      "        from setup_posix import get_config\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_66562a10f1fb4d89acbe3d321844b8cd/setup_posix.py\", line 2, in <module>\n",
      "        from ConfigParser import SafeConfigParser\n",
      "    ModuleNotFoundError: No module named 'ConfigParser'\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/a5/e9/51b544da85a36a68debe7a7091f068d802fc515a3a202652828c73453cad/MySQL-python-1.2.5.zip#sha256=811040b647e5d5686f84db415efd697e6250008b112b6909ba77ac059e140c74 (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading MySQL-python-1.2.4.zip (113 kB)\n",
      "\u001b[K     |████████████████████████████████| 113 kB 6.5 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-7lk_m5hy\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/\n",
      "    Complete output (31 lines):\n",
      "    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.28.tar.gz\n",
      "    Traceback (most recent call last):\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/distribute_setup.py\", line 143, in use_setuptools\n",
      "        raise ImportError\n",
      "    ImportError\n",
      "    \n",
      "    During handling of the above exception, another exception occurred:\n",
      "    \n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/setup.py\", line 7, in <module>\n",
      "        use_setuptools()\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/distribute_setup.py\", line 145, in use_setuptools\n",
      "        return _do_download(version, download_base, to_dir, download_delay)\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/distribute_setup.py\", line 123, in _do_download\n",
      "        tarball = download_setuptools(version, download_base,\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_6a9d656687694c9689b4002cda0515e2/distribute_setup.py\", line 194, in download_setuptools\n",
      "        src = urlopen(url)\n",
      "      File \"/Users/ajithmurali/opt/anaconda3/lib/python3.9/urllib/request.py\", line 214, in urlopen\n",
      "        return opener.open(url, data, timeout)\n",
      "      File \"/Users/ajithmurali/opt/anaconda3/lib/python3.9/urllib/request.py\", line 523, in open\n",
      "        response = meth(req, response)\n",
      "      File \"/Users/ajithmurali/opt/anaconda3/lib/python3.9/urllib/request.py\", line 632, in http_response\n",
      "        response = self.parent.error(\n",
      "      File \"/Users/ajithmurali/opt/anaconda3/lib/python3.9/urllib/request.py\", line 561, in error\n",
      "        return self._call_chain(*args)\n",
      "      File \"/Users/ajithmurali/opt/anaconda3/lib/python3.9/urllib/request.py\", line 494, in _call_chain\n",
      "        result = func(*args)\n",
      "      File \"/Users/ajithmurali/opt/anaconda3/lib/python3.9/urllib/request.py\", line 641, in http_error_default\n",
      "        raise HTTPError(req.full_url, code, msg, hdrs, fp)\n",
      "    urllib.error.HTTPError: HTTP Error 403: SSL is required\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/90/5a/ce7bef80825c2188cf507baf57b37516e18dffdb198a6766a597f703059a/MySQL-python-1.2.4.zip#sha256=e405f9d6be33923d428acaa4db4f4470427f1d15ea0d2d82a933449ace26bbd9 (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading MySQL-python-1.2.3.tar.gz (70 kB)\n",
      "\u001b[K     |████████████████████████████████| 70 kB 9.9 MB/s  eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /Users/ajithmurali/opt/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_8416a65803494722831b33649a60b7bd/setup.py'\"'\"'; __file__='\"'\"'/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_8416a65803494722831b33649a60b7bd/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-pip-egg-info-cawpnyqn\n",
      "         cwd: /private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_8416a65803494722831b33649a60b7bd/\n",
      "    Complete output (8 lines):\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_8416a65803494722831b33649a60b7bd/setup.py\", line 11, in <module>\n",
      "        from setup_posix import get_config\n",
      "      File \"/private/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/pip-install-w6kb5p14/mysql-python_8416a65803494722831b33649a60b7bd/setup_posix.py\", line 101\n",
      "        print \"\"\"You shouldn't be running this directly; it is used by setup.py.\"\"\"\n",
      "              ^\n",
      "    SyntaxError: Missing parentheses in call to 'print'. Did you mean print(\"\"\"You shouldn't be running this directly)?\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/9a/81/924d6799494cf7fb24370335c2f782088d6ac4f79e4137d4df94cea3582c/MySQL-python-1.2.3.tar.gz#sha256=7de66fbbf923634e7c965aeaefa74642ba75ae20ee1cefcefc3009595b7a7e6e (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[31mERROR: Could not find a version that satisfies the requirement MySQL-python (from versions: 1.2.3b1, 1.2.3b2, 1.2.3rc1, 1.2.3, 1.2.4b1, 1.2.4b2, 1.2.4b3, 1.2.4b4, 1.2.4b5, 1.2.4rc1, 1.2.4, 1.2.5)\u001b[0m\n",
      "\u001b[31mERROR: No matching distribution found for MySQL-python\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25hNote: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install MySQL-python"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "155cb805",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: pyodbc in ./opt/anaconda3/lib/python3.9/site-packages (4.0.0-unsupported)\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install pyodbc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "b88502c3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting mysql-connector-python\n",
      "  Downloading mysql_connector_python-8.0.28-py2.py3-none-any.whl (342 kB)\n",
      "\u001b[K     |████████████████████████████████| 342 kB 3.7 MB/s eta 0:00:01\n",
      "\u001b[?25hCollecting protobuf>=3.0.0\n",
      "  Downloading protobuf-3.19.4-cp39-cp39-macosx_10_9_x86_64.whl (961 kB)\n",
      "\u001b[K     |████████████████████████████████| 961 kB 11.8 MB/s eta 0:00:01\n",
      "\u001b[?25hInstalling collected packages: protobuf, mysql-connector-python\n",
      "Successfully installed mysql-connector-python-8.0.28 protobuf-3.19.4\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install mysql-connector-python"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "7e07f404",
   "metadata": {},
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'mysqldb'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "\u001b[0;32m/var/folders/56/tgrtw5g17z9006b9fnq9tq640000gp/T/ipykernel_14623/3635074796.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mimport\u001b[0m \u001b[0mmysqldb\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mmydb\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mmysql\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mconnector\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m mydb = mysql.connector.connect(\n\u001b[1;32m      5\u001b[0m   \u001b[0mhost\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"127.0.0.1\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'mysqldb'"
     ]
    }
   ],
   "source": [
    "import PyMySQL as pymy\n",
    "import mysql.connector\n",
    "\n",
    "pymql = mysql.connector.connect(\n",
    "  host=\"127.0.0.1\",\n",
    "  user=\"root\",\n",
    "  password=\"Ajithkumar@1\",\n",
    "  database=\"mydatabase\"\n",
    ")\n",
    "\n",
    "mycursor = mydb.cursor()\n",
    "\n",
    "sql = \"INSERT INTO order1 (order_data,order_id,product_id,customer_id,quentity,amount) VALUES (%s, %s, %s, %s,%s,%s)\"\n",
    "val = (40,1004,4,2004,301,100)\n",
    "mycursor.execute(sql, val)\n",
    "\n",
    "mydb.commit()\n",
    "\n",
    "print(mycursor.rowcount, \"record inserted.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ac0318e2",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "print(\"products are:p1=10,p2=11,p3=12\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dd25afcb",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "652c0225",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "order_data:100\n",
      "order_id:10\n",
      "product_id:1000\n",
      "customer_id:2000\n",
      "quentity:30000\n",
      "amount:400\n",
      "Data entered successfully.\n",
      "\n",
      "The SQLite connection is closed.\n"
     ]
    }
   ],
   "source": [
    "import  sqlite3\n",
    "conn  =  sqlite3 . connect ( 'mydatabase.db' )\n",
    "cursor  =  conn.cursor ()\n",
    "cursor.execute(\"CREATE TABLE order2(order_data int(5), order_id int(5), product_id int(5), customer_id int(5),quentity int(5),amount int(5));\")\n",
    "s_data = input('order_data:')\n",
    "s_id = input('order_id:')\n",
    "s_pid = input('product_id:')\n",
    "s_cuid = input('customer_id:')\n",
    "s_quen = input('quentity:')\n",
    "s_amount = input('amount:')\n",
    "cursor.execute(\"\"\"\n",
    "INSERT INTO order2(order_data, order_id, product_id, customer_id,quentity,amount)\n",
    "VALUES (?,?,?,?,?,?)\n",
    "\"\"\", (s_data, s_id, s_pid, s_cuid, s_quen, s_amount))\n",
    "conn.commit ()\n",
    "print ( 'Data entered successfully.' )\n",
    "conn . close ()\n",
    "if (conn):\n",
    "    conn.close()\n",
    "    print(\"\\nThe SQLite connection is closed.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "686dec46",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "caab7992",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "de01eeec",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "70714789",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f509a164",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "833c4c23",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c880060a",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fd3084cd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "77da2b71",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
